app/controllers/automated_tests_controller.rb

```diff
@@ -48,11 +48,25 @@ def student_interface
 
   def execute_test_run
     assignment = Assignment.find(params[:assignment_id])
+
+    # If no test groups can be run by students, flash appropriate message and return early
+    test_group_settings = TestGroup.where(assessment_id: assignment.id).pluck(:autotest_settings)
+    # rubocop:disable Rails/Pluck
+    test_group_categories = test_group_settings.map { |settings| settings['category'] }
```

Comment:
You may not be aware that there are two different `pluck` methods. The one rubocop is telling you to use is https://api.rubyonrails.org/classes/Enumerable.html#method-i-pluck